doc/Technical Documentation/Compiler Intermediate Language format.md

```diff
@@ -166,16 +166,16 @@ Finally, instructions like this:
 ```
 RV.T3.I =   #0.I
 ```
-instruct the compiler to generate the sequence required for a return value, e.g. on the x86 most integers would be moved into eax.
+instruct the compiler to generate the sequence required for a return value, e.g. on the x86 most integers would be moved into `EAX`.
 Not shown here you can also have something similar after a `GOSUB` statement:
 ```
 GOSUB  #_compute_value:PC.UI
 PARMADJ #0.I
 T4.I = RV.T3.I
 ```
-In this case the RV refers to the value returned by the function, with the assumption on the x86 that T3 would be in eax.
+In this case the RV refers to the value returned by the function, with the assumption on the x86 that T3 would be in `EAX`.
 
-There are other registers on the x86 that are predefined, for example EAX/EDX are used in division and some forms of multiplication.   
+There are other registers on the x86 that are predefined, for example `EAX/EDX` are used in division and some forms of multiplication.   
```

Comment:
Just to tease you: 

```diff
-`EAX/EDX`
+`EAX`/`EDX`
```

Reply:
Eh, I had them joined here because they are being referred to as the 64 bit operation register pair instead of the individual registers themselves and I wanted to make the distinction.